docs/proposals/pepxxx_wheel_variant_support.md

```diff
@@ -527,12 +528,12 @@ supported but has the lowest priority among wheel variants, while being preferab
 - **Variant Provider**: A provider of supported and valid variant properties for a specific namespace, usually
 in the form of a Python package that implements system detection.
 
-- **Install-time Provider (Plugin)**: A provider in the form of a plugin that is queried while installing the wheel.
+- **Install-time Provider (Plugin)**: A provider implemented as a plugin that can be queried during wheel installation.
 
 - **Ahead-of-Time Provider**: A provider that features a static list of supported properties which is then embedded
```

Comment:
TODO: Here we would need to add use cases when you use ``Install-time Provider`` vs ``Ahead-of-Time Provider`` what are pro's and cons of each. What if they are used togeather ?

Reply:
Examples are in the "rationale" section.

Reply:
Basically what I was thinking of : Ahead-of-Time Provider - suggested use in these use cases. Install-time Provider - following use cases. And how the system would behave in each case step by step. What are security concerns for each use case.